### ISSUE TITLE
Breakage in Rust 1.15 Beta

### ISSUE THREAD

**Issue by johnthagen** [PistonDevelopers/gfx_graphics#317]
See: https://travis-ci.org/johnthagen/rust-belt/jobs/185932707

```
error[E0432]: unresolved import ``
  --> /home/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/piston2d-gfx_graphics-0.33.1/src/back_end.rs:30:1
   |
30 |   gfx_pipeline_base!( pipe_colored {
   |  _^ starting here...
31 | |     pos: ::gfx::VertexBuffer<PositionFormat>,
32 | |     color: ::gfx::VertexBuffer<ColorFormat>,
33 | |     blend_target: ::gfx::BlendTarget<::gfx::format::Srgba8>,
34 | |     stencil_target: ::gfx::StencilTarget<::gfx::format::DepthStencil>,
35 | |     blend_ref: ::gfx::BlendRef,
36 | |     scissor: ::gfx::Scissor,
37 | | });
   | |___^ ...ending here: no `` in the root
   |
   = note: this error originates in a macro outside of the current crate
error[E0432]: unresolved import ``
  --> /home/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/piston2d-gfx_graphics-0.33.1/src/back_end.rs:39:1
   |
39 | gfx_pipeline_base!( pipe_textured {
   | ^ no `` in the root
   |
   = note: this error originates in a macro outside of the current crate
error: aborting due to 2 previous errors
Build failed, waiting for other jobs to finish...
error: Could not compile `piston2d-gfx_graphics`.
```

My `Cargo.toml`:

```toml
[dependencies]
piston_window = "0.59.0"
piston2d-opengl_graphics = "0.36.1"
piston-music = "0.16.0"
```

It could be that piston_window needs to pin to a newer version of `gfx_graphics`?

**Comment by kvark** [comment]
`gfx_graphics` currently depends on `gfx-0.12.0`. This is issue is fixed in `gfx-0.12.2`

**Comment by kvark** [comment]
So you could try `cargo update` and see if that picks up gfx-0.12.2 automatically for you.

**Comment by kvark** [comment]
@johnthagen the new versions of piston_window, gfx_texture, and gfx_graphics are now published, based on gfx-0.13. Let me know if you need anything else!

**Comment by johnthagen** [comment]
[Fixed](https://travis-ci.org/johnthagen/rust-belt), thanks!
